pkg/kubelet/server/stats/summary.go

```diff
@@ -99,7 +99,7 @@ func (sp *summaryProviderImpl) Get(updateStats bool) (*statsapi.Summary, error)
 		}
 		s, _, err := sp.provider.GetCgroupStats(cont.name, cont.forceStatsUpdate)
 		if err != nil {
-			glog.Errorf("Failed to get system container stats for %q: %v", cont.name, err)
+			glog.V(5).Errorf("Failed to get system container stats for %q: %v", cont.name, err)
```

Comment:
Seems like a throttle would be more useful than turning down the verbosity. Do we have a standard library for throttle log lines?

Reply:
> I'm a bit confused by the motivation for this. Why is cAdvisor running with docker_only in the kubelet?

The reason for this is that with newer Debian versions, systemd is used and everything is in a cgroup. When used with legacy kubelet source for heapster, we will scrape more cgroup data. With `docker-only` we can dramatically reduce the data size heapster will scrape for a large cluster.

Reply:
> Seems like a throttle would be more useful than turning down the verbosity. Do we have a standard library for throttle log lines?

@tallclair I am afraid glog can not meet the demands.  We should make kubelet stats api worable with setting `docker-only`.

Reply:
Yeah, OK, this is fine as a stop gap. Can you file an issues for making the Kubelet stats API compatible with the docker-only setting, and reference that issue from a comment here?